orb/schema/validator.py

```diff
@@ -106,7 +106,7 @@ def validate(self, column, value):
 
         :return     <bool>
         """
-        if type(value) != bool and not bool(value):
+        if type(value) not in [bool, int, float] and not bool(value):
```

Comment:
So... if the goal is to check that the values are not "NULL", there are really only two such null values in Python and the one of them is sometimes a valid value for a database row.

#1 `None`
#2 `''`    (empty string)

Should this check simply be:

`if value is None:`

or, possibly:

`if value is None or value == '':`